Src/AutoFixture.NUnit3.UnitTest/InlineAutoDataAttributeTest.cs

```diff
@@ -0,0 +1,36 @@
+﻿using NUnit.Framework;
+
+namespace Ploeh.AutoFixture.NUnit3.UnitTest
+{
+    [TestFixture]
+    public class InlineAutoDataAttributeTest
+    {
+        [Test]
+        public void InlineAutoDataInheritsAutoData()
+        {
+            Assert.That(new InlineAutoDataAttribute(), 
+                Is.AssignableTo(typeof(AutoDataAttribute)));
+        }
+
+        [Test]
+        public void InlineAutoDataCanBeExtendedWithImplementationOfIFixture()
+        {
+            var extended = new InlineAutoDataAttributeStub();
+
+            Assert.That(extended, Is.AssignableTo(typeof(AutoDataAttribute)));
+        }
+
+        private class InlineAutoDataAttributeStub : InlineAutoDataAttribute
```

Comment:
AFAICT, this test is here in order to prevent someone, in the future, from removing the `.ctor` that takes an `IFixture` instance as parameter.

If this is the case, I'm wondering if *DerivedInlineAutoDataAttribute* would be a slightly better name instead of InlineAutoDataAttributeStub.


What do you think? :smiley: